include/drivers/lora.h

```diff
@@ -141,9 +141,7 @@ static inline int lora_send(const struct device *dev,
  * @param data      Buffer to hold received data
  * @param size      Size of the buffer to hold the received data. Max size
 		    allowed is 255.
- * @param timeout   Timeout value in milliseconds. API also accepts, 0
-		    for no wait time and SYS_FOREVER_MS for blocking until
-		    data arrives.
+ * @param timeout   Duration to wait for a packet.
```

Comment:
We should also brief the use case of `K_NO_WAIT` and `K_FOREVER` as done before.

Reply:
Why is this needed? The behavior is self evident from the arguments.
`K_NO_WAIT` == don't wait at all (will never actually receive anything) 
`K_FOREVER` == wait forever
I don't see other functions documenting the meaning of `K_NO_WAIT` and `K_FOREVER`.

Reply:
It'd be helpful for the newcomers to Zephyr as they don't need to go through the `k_timeout_t` API doc.

Reply:
`k_timeout_t` is a core feature of Zephyr. Users should already be familiar with it, and if they're not, going to the struct definition is going to be far more useful to them than giving them two magic values.
From my perspective its equivalent to the following:
```
/**
 * @param a Integer to add.
 *          Adds 2 to b if the value is 2.
 *          Adds 1000 to b if the value is 1000.
 */
int add(int a, int b) { return a + b; };
```
Why specify the values 2 and 1000? Are these the only two values that work? Is the behavior of other integers different?
No, they all work, don't imply they're any different.

Reply:
No strong feeling here... hence resolving.